api/src/legacy_soap_api/applicants/schemas/__init__.py

```diff
@@ -0,0 +1,17 @@
+from src.legacy_soap_api.applicants.schemas.cfda_details import CFDADetails
+from src.legacy_soap_api.applicants.schemas.get_opportunity_list_request import (
+    GetOpportunityListRequest,
+)
+from src.legacy_soap_api.applicants.schemas.get_opportunity_list_response import (
+    GetOpportunityListResponse,
+)
+from src.legacy_soap_api.applicants.schemas.opportunity_details import OpportunityDetails
+from src.legacy_soap_api.applicants.schemas.opportunity_filter import OpportunityFilter
```

Comment:
Since we're going to have a massive amount of these schemas as we build out more endpoints, I'd actually say it's fine if we put more of the schemas in one file - something like a file per route, plus a few schemas might live in "shared" schema files.